.expeditor/buildkite/verify.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+#
+# This script runs a passed in command, but first setups up the bundler caching on the repo
```

Comment:
```suggestion
```

This comment has a typo, but is also not accurate after this change.

Reply:
Yeah, i just removed the comment all together. Thanks!